xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/comments2.vm

```diff
@@ -21,10 +21,34 @@
 #set($showcomments = $xwiki.getSpacePreference('showcomments'))
 #end
 #if($showcomments!="no")
-#if($xwiki.getSpacePreferenceAsInt('commentsorder', 1) == 0)
 #set($comments = $doc.getComments())
+#set ($commentMap = {})
+#set ($keysWithMultipleComments = $collectiontool.getSet())
+#foreach ($comment in $comments)
```

Comment:
Is there a reason to avoid indentation here?

Reply:
I followed the style of the file: comments2.vm is used in pdf.vm, and I imagine that any space is displayed in the output. It makes me realize that I wrongly kept spaces after `set` I should remove those too probably.